main.sh

```diff
@@ -249,23 +262,23 @@ fi
 ##############################################################################
 
 # debug mode
-if [ "${arg_d}" = "1" ]; then
+if [ "${arg_d:?}" = "1" ]; then
   set -o xtrace
   LOG_LEVEL="7"
 fi
 
 # verbose mode
-if [ "${arg_v}" = "1" ]; then
+if [ "${arg_v:?}" = "1" ]; then
   set -o verbose
 fi
 
 # no color mode
-if [ "${arg_n}" = "1" ]; then
+if [ "${arg_n:?}" = "1" ]; then
   NO_COLOR="true"
 fi
 
 # help mode
-if [ "${arg_h}" = "1" ]; then
+if [ "${arg_h:?}" = "1" ]; then
```

Comment:
interesting approach... I was going to have just used disable=... but I think I like this more